docs/changelog.md

```diff
@@ -53,7 +53,7 @@ We use _breaking :warning:_ to mark changes that are not backward compatible \(r
 
 ### Added
 
-* [\#321](https://github.com/tellor-io/telliot/pull/363) `interval` field in the `indexes.json` file. This sets a custom trackerCycle for a specific \(e.g. an `interval: 600` would lead to the API being updated every hour\)
+* [\#321](https://github.com/tellor-io/telliot/pull/363) `interval` field in the `api.json` file. This sets a custom trackerCycle for a specific \(e.g. an `interval: 600` would lead to the API being updated every hour\)
```

Comment:
```suggestion
* [\#321](https://github.com/tellor-io/telliot/pull/363) `interval` field in the `indexes.json` file. This sets a custom trackerCycle for a specific \(e.g. an `interval: 600` would lead to the API being updated every hour\)
```